docs/04-operating-scs/08-backups/user_data_backups.md

```diff
@@ -0,0 +1,479 @@
+
+See the [corresponding appendix section](#image-creation-action-for-servers-with-attached-volumes) for further details.
+
+:::
+
+Ephemeral Storage disks of virtual machines can be backed up to Glance images easily by using the `createImage` Compute API action or the corresponding client command:
+
+```bash
+openstack server image create --name $IMAGE_NAME $SERVER_NAME_OR_ID
+```
+
+This will create a Glance image containing a one-to-one copy of the data on the Ephemeral Storage disk at the time of execution.
+
+If the necessity arises to store this backup outside of the IaaS infrastructure, the download procedure as described in [Image backup using download](#image-backup-using-download) may be used after the image creation.
+
+## Volume data backup using Cinder Backup API
+
+The following instructions only apply if the infrastructure offers the Cinder Backup API.
+
+:::note
+
+Backups of volumes using a volume type that uses encryption will retain their encryption and a clone of the original encryption key is created in Barbican linked to the backup.
+These backups can only be restored when the Barbican service is available and still has the corresponding copy of the encryption key.
+
+Also, it is advised to take note of the exact volume type when creating a backup of an encrypted volume, because this information will be needed to restore the backup.
+See [restoring an encrypted volume backup](#restoring-an-encrypted-volume-backup-cinder-backup).
+
+:::
+
+:::info
+
+It might be difficult or even impossible for a user to transfer backups created by the Cinder Backup API outside of the IaaS infrastructure, depending on the backup backend.
+A more easily accessible backup of volumes can be created by using Glance images.
+See the [section about volume data backup using Glance images](#volume-data-backup-using-glance-images) for details.
+
+:::
+
+### Backup of detached volumes
+
+Backups can be created using the Cinder Backup API or the corresponding client commands:
+
+```bash
+openstack volume backup create $VOLUME_NAME_OR_ID
+```
+
+Further backups of the same volume can subsequently be created as incremental backups using the following command:
+
+```bash
+openstack volume backup create --incremental $VOLUME_NAME_OR_ID
+```
+
+### Backup of attached volumes
+
+:::note
+
+When creating backups of attached (in-use) volumes, the `force` parameter has to be used. These backups capture a state of the full volume at runtime. They will be crash-consistent but not app-consistent.
+
+:::
+
+In case of attached (in-use) volumes, backups can only be created while also specfiying the `force` parameter:
```

Comment:
This sentence is a repetition of the sentence above. I would re-write it. e.g. as follows:
```suggestion
The API or the client may be used to backup attached volume,  for example:
```

Reply:
I rephrased the note banner instead.